install.sh

```
@@ -1129,54 +1250,71 @@
+                        while read -r file; do
+                            error_list="${error_list}  • $file\n"
+                        done < <(find "$dir" -type f ! -writable 2>/dev/null)
+                    fi
+                done
+            }
+        }
+        
+        # Show error list if there were problems
+        if [ -n "$error_list" ]; then
+            print_message "The following files could not be removed:" "$RED"
+            printf "$error_list"
```

Shellcheck (0.10.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)